docs/src/content/docs/es/environmental-impact.md

```diff
@@ -73,20 +73,21 @@ Elegir un proveedor de alojamiento que utilice energía renovable significa tene
 ¿Curioso por saber cómo se comparan otros frameworks de documentación?
 Estas pruebas con el [Calculadora de Carbono de Sitios Web][wcc] comparan páginas similares construidas con diferentes herramientas.
 
-| Framework                   | CO₂ por visita a la página |
-| --------------------------- | -------------------------- |
-| [Starlight][sl-carbon]      | 0.01g                      |
-| [VitePress][vp-carbon]      | 0.05g                      |
-| [Docus][dc-carbon]          | 0.05g                      |
-| [Sphinx][sx-carbon]         | 0.07g                      |
-| [MkDocs][mk-carbon]         | 0.10g                      |
-| [Nextra][nx-carbon]         | 0.11g                      |
-| [docsify][dy-carbon]        | 0.11g                      |
-| [Docusaurus][ds-carbon]     | 0.24g                      |
-| [Read the Docs][rtd-carbon] | 0.24g                      |
-| [GitBook][gb-carbon]        | 0.71g                      |
-
-<small>Datos recopilados el 14 de mayo de 2023. Haz clic en un enlace para ver cifras actualizadas.</small>
+| Framework                   | CO₂ per page visit | Rating |
```

Comment:
Should the headings here be translated?

“CO₂ per page visit” was “CO₂ por visita a la página” originally and I guess there’s a translation for “Rating” too?

Reply:
I forgot about those, let me fix it real quick 😁